examples/management-cli/pom.xml

```diff
@@ -19,7 +19,7 @@
         <dependency>
             <groupId>com.descope</groupId>
             <artifactId>java-sdk</artifactId>
-            <version>1.0.8</version>
+            <version>1.0.10</version>
```

Comment:
why 2 jumps ?

Reply:
I forgot to update in the previous release - this is the example